index.js

```diff
@@ -62,15 +62,18 @@ Throttle.prototype.rateLimit = function (key, cb) {
 
   var burst = self.burst
   var rate = self.rate
+  var window = self.window
```

Comment:
Probably should avoid the variable name `window` if just for the fact that most syntax highlighters will treat it as the global `window` object.